launch/tier4_perception_launch/launch/perception.launch.xml

```diff
@@ -88,15 +90,15 @@
     description="options: `traffic_light_classifier_mobilenetv2_batch_*` or `traffic_light_classifier_efficientNet_b1_batch_*`. The batch number must be either one of 1, 4, 6"
   />
 
-  <!-- Camera Lidar Fusion Parameters -->
+  <!-- Camera-Lidar Fusion Parameters -->
```

Comment:
How about the following to be consistent with the others?
```suggestion
  <!-- Camera-Lidar fusion param -->
```

Reply:
I also fixed at [23cb2f1](https://github.com/autowarefoundation/autoware.universe/pull/4995/commits/23cb2f1119f70e5ac0dc172749e9b1ee7907944e)